frontend/src/components/Dashboard/DashboardResultsTable.tsx

```diff
@@ -67,10 +85,10 @@ const DashboardResultsTable = (props: any) => {
     const tableHeadContent = (results: any) => {
         return results[0]?.exercises.length > 0 ? (
             <thead
-                className="text-gray-800 bg-gray-100 rounded-lg"
+                className="w-full text-gray-800 bg-gray-100 rounded-lg" style={{ tableLayout: "auto" }}
```

Comment:
`table-layout:"auto"` exists in Tailwind : 
```suggestion
                className="w-full text-gray-800 bg-gray-100 rounded-lg table-auto"
```

Reply:
Done